src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -192,10 +194,13 @@
     to get available permissions for Microsoft Graph API, run `az ad sp show --id 00000003-0000-0000-c000-000000000000`.
     Application permissions under the `appRoles` property correspond to `Role` in --api-permissions.
     Delegated permissions under the `oauth2Permissions` property correspond to `Scope` in --api-permissions.
+
+
+    For details on Microsoft Graph permissions, see https://learn.microsoft.com/graph/permissions-reference
 examples:
-  - name: Add Microsoft Graph delegated permission User.Read (Sign in and read user profile).
+  - name: Add Microsoft Graph delegated permission User.Read
```

Comment:
```suggestion
  - name: Add Microsoft Graph delegated permission `User.Read`.
```

Reply:
As shown in https://learn.microsoft.com/en-us/graph/permissions-reference, Microsoft Graph permissions are usually not marked as `code`.